config/systems.php

```diff
@@ -425,28 +425,29 @@
         'active' => false,
     ],
     61 => [
-        'name' => 'N-Gage',
+        'name' => 'Nokia N-Gage',
         'name_full' => 'Nokia N-Gage',
         'name_short' => 'N-Gage',
         'manufacturer' => 'Nokia',
         'active' => false,
     ],
     62 => [
-        'name' => '3DS',
+        // 'name' => '3DS',
```

Comment:
leftover comment should be removed

Reply:
thanks! https://github.com/RetroAchievements/RAWeb/pull/2172/commits/cdd2d9f383eae7b445b317e8503764b1ad86845e